themes/bootstrap3/templates/search/results.phtml

```diff
@@ -38,6 +38,7 @@
   if ($this->params->getOptions()->getListViewOption() != "full") {
     $this->headScript()->appendFile("record.js");
     $this->headScript()->appendFile("embedded_record.js");
+    $this->headScript()->appendFile("truncate.js");
```

Comment:
Here's another instance of a conditional load of truncate.js. This one is also causing a conflict with the latest dev branch, because some of the script loading got refactored to https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/templates/search/results-scripts.phtml -- should we actually be including this there instead?